src/Types/ConversionException.php

```diff
@@ -107,7 +109,7 @@ public static function conversionFailedSerialization($value, $format, $error)
             $actualType,
             $format,
             $error
-        ));
+        ), 0, func_num_args() >= 4 ? func_get_arg(3) : null);
```

Comment:
It looks like this method isn't covered by any tests. Would it be possible to add some? Additionally, the `conversionFailedUnserialization()` could use the same approach and accept a previous exception. Otherwise, seemingly similar methods now have a different signature.

Reply:
I've added a test.

Reply:
I can add the parameter to `conversionFailedUnserialization()` on 4.0.x. Right now, the DBAL codebase wouldn't need to pass a previous exception here.

Reply:
I see, it's not used for JSON deserialization. Unless this new parameter could be used by the DBAL, there's no point in adding it.